ui-v2/app/templates/components/tab-nav.hbs

```diff
@@ -1,9 +1,13 @@
 {{!<nav role="tablist">}}
     <ul>
 {{#each items as |item|}}
-        <li class={{if (eq selected (slugify item)) 'selected' }}>
-            <label for="radiogroup_{{name}}_{{slugify item}}" data-test-radiobutton="{{name}}_{{slugify item}}" role="tab" aria-controls="radiogroup_{{name}}_{{slugify item}}_panel">
-                <a>{{item}}</a>
+        <li class={{if (eq selected (if item.label (slugify item.label) (slugify item))) 'selected'}}>
+            <label for="radiogroup_{{name}}_{{if item.label (slugify item.label) (slugify item)}}" data-test-radiobutton="{{name}}_{{if item.label (slugify item.label) (slugify item)}}" role="tab" aria-controls="radiogroup_{{name}}_{{if item.label (slugify item.label) (slugify item)}}_panel">
+{{#if item.href }}
+  <a href="{{ item.href }}">{{ item.label }}</a>
+{{ else }}
+  <a>{{ item }}</a>
```

Comment:
Just 're-seen' this. Gonna come back to this for a later date, the attribute-less <a> isn't in this PR (if you look at the removed code it was there also), but it could do with re-looking at. I vaguely remember this being something to do with being able to have a tab bar that changes without changing the URL or any state, or having a tab bar where the tabs act as links - also I think this makes the tabs more accessible in that they are automatically in the tablist when you tab through the page. I've added this to my todo to come back later.